app/code/Magento/Catalog/Model/ImageUploader.php

```diff
@@ -67,14 +67,9 @@ class ImageUploader
     /**
      * List of allowed image mime types
      *
-     * @var array
+     * @var string[]
      */
-    private $allowedMimeTypes = [
-        'image/jpg',
-        'image/jpeg',
-        'image/gif',
-        'image/png',
-    ];
+    protected $allowedMimeTypes;
```

Comment:
Must remain private.

Reply:
it can be private.
what is the purpose of users can extend allowedExtensions but allowedMimeTypes is hardcoded?
I think two property is connected logically.